domains/furret.json

```diff
@@ -4,6 +4,6 @@
         "email": "furret@transistorcafe.net"
     },
     "record": {
-        "CNAME": "furret.tplinkdns.com"
+        "CNAME": "walccdev.github.io"
```

Comment:
Nothing is displayed.

Reply:
its a blank page, making the site

Reply:
When the site is finished, you may open a pull request and then it will get approved.

Reply:
finished